assembl/graphql/user.py

```diff
@@ -172,13 +172,6 @@ def mutate(root, args, context, info):
                 if old_password == new_password:
                     raise Exception(u"004: The new password has to be different than the current password.")
 
-                from ..auth.password import verify_password
```

Comment:
No, do not remove this logic. It's the check we make for the new password vs all of the old passwords

Reply:
This check is already done in `validate_password` in models/auth.py. This is duplicated, and I think the right place is in `validate_password`, which is called by `password_p`

Reply:
I see what you mean. Very well. I have added a new commit for an extra check that was done here, to be done in the downstream.